internal/cephfs/util.go

```diff
@@ -34,7 +34,10 @@ import (
 type volumeID string
 
 func execCommandErr(ctx context.Context, program string, args ...string) error {
-	_, _, err := util.ExecCommand(ctx, program, args...)
+	_, stdErr, err := util.ExecCommand(ctx, program, args...)
```

Comment:
How this function was working earlier for other cases?

can you check when we will get an error in stdErr and err? we have e2e tests for negative scenarios also how it was passing earlier?

Reply:
It was passing earlier as it was always returning the [default error](https://github.com/ceph/ceph-csi/blob/master/internal/cephfs/volume.go#L241) in place of specialized (eg: `Error ENOTEMPTY`)